.gitlab-ci.yml

```diff
@@ -391,6 +391,7 @@ test-linux-stable-int:
     paths:
       - ${CI_COMMIT_SHORT_SHA}_int_failure.log
 
+# TODO: (dp) Not 100% sure what this does; maybe just rename the CI stage?
 check-web-wasm:
```

Comment:
AFAIU it's a set of tests combined into one job so that reuse the cache better. FWIW now I'd rename it to `check-wasm`

Reply:
Thing is this PR removes compiling the substrate client to wasm (obviously the runtime remains in wasm) so that's not a great name. I'll go with `check-tracing` for now, let me know if you prefer something else!